### ISSUE TITLE
Add example DMPs to landing page and Reference dropdown menu

### ISSUE THREAD

**Issue by shilowil** [portagenetwork/roadmap#489]
We are looking to make example DMPs easier to find by adding them to the platform. There are two places we would like to add them (open to suggestions!):

1. Landing page: beneath "Getting started" and after "Primer - Data Management Plan."
2. In the "Reference" dropdown menu visible to logged in users.

Choosing two places because the landing page is not visible to logged in users while the "Reference" menu is not visible to logged out users. 

Will follow up with the link that should be used. Tagging @lagoan for triage

**Comment by lagoan** [comment]
@isawil should we prioritize this over our upgrade to version 4 ?

**Comment by shilowil** [comment]
@lagoan No, let's do this after the upgrade to 4 (assuming the upgrade to 4 is coming in the next few weeks)
